components/dialog/dialog-mixin.js

```diff
@@ -93,14 +101,18 @@ export const DialogMixin = superclass => class extends RtlMixin(superclass) {
 	async updated(changedProperties) {
 		super.updated(changedProperties);
 		if (!changedProperties.has('opened')) return;
+
+		let dialogService;
```

Comment:
Could this just be...
```javascript
const dialogService = await getIfrauDialogService();
if (this.opened) {
    if (dialogService) {
        ...
    }
    this._open();
} else {
    if (dialogService) {
        ...
    }
    this._close();

}
```

Reply:
We probably would need to update the helper to include.
```javascript
if (!window.ifrauclient) return;
```

Reply:
The helper already checks `window.ifrauclient`, so we're good there, but I think we might want to switch the order of the checks just in case (somehow) the service gets set without ifrauclient loaded.

Reply:
Ah, but then if it gets set without ifrauclient loaded, it would have some value so we can't just return it.